content/images/content-image.svg

```diff
@@ -0,0 +1 @@
+asdasdasd
```

Comment:
The problem is that git doesn't add folders if no image is there (and I do think having the folder is nice as boilerplate). Should I add a real image (like a ginetta logo)?